src/main/java/com/itacademy/cms/controller/CategoryController.java

```diff
@@ -1,7 +1,7 @@
 package com.itacademy.cms.controller;
 
 import com.itacademy.cms.exeption.CategoryNotFoundException;
-import com.itacademy.cms.mapper.EntityMapper;
+import com.itacademy.cms.mapper.MapStructMapper;
```

Comment:
We seem to have agreed to name Mapper as EntityMapper

Reply:
it look like i have miss it

Reply:
No problem, it can be MapStructMapper either